CONTRIBUTING.md

```diff
@@ -159,6 +161,66 @@ ktlint --format
 
 <a id="issues-projects"></a>
 
+### Unit Testing Conventions for Scribe Android
+
+To maintain consistency and clarity in the Scribe Android codebase, we recommend you to follow these conventions when writing unit tests. These guidelines cover the organization, naming conventions, scope, and structure for unit tests.
+
+#### 1. Project Structure for Unit Tests
+
+- **Location**: Place all unit tests in the `src/test/java` directory, mirroring the structure of the `src/main/java` directory. For new classes or features, ensure their corresponding test classes follow the same package structure.
```

Comment:
This must be an Android project structure thing - I'm newer to Android - why is kotlin code put under java directories?

The unit tests I added as an example are under src/test/kotlin so a follow up will need to be moving the example to the correct structure if src/test/java is the desired structure.

Reply:
I'm not actually sure on this either, @albendz :) Was the structure of the project that was forked. Maybe we could look into this in #223 where some renaming will happen 🤔